mllib/src/main/scala/org/apache/spark/ml/feature/MinHash.scala

```diff
@@ -32,13 +32,7 @@ import org.apache.spark.sql.types.StructType
  * :: Experimental ::
  *
  * Model produced by [[MinHash]], where multiple hash functions are stored. Each hash function is
- * a perfect hash function:
- *    `h_i(x) = (x * k_i mod prime) mod numEntries`
- * where `k_i` is the i-th coefficient, and both `x` and `k_i` are from `Z_prime^*`
- *
- * Reference:
- * [[https://en.wikipedia.org/wiki/Perfect_hash_function Wikipedia on Perfect Hash Function]]
- *
+ * a perfect hash.
```

Comment:
@jkbradley is it not more confusing to not have any reference or further explanation here? You mentioned in https://github.com/apache/spark/pull/15148#issuecomment-258991190 to remove this, but should we not have some doc here or a better reference instead of nothing?